WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/CardReaderSettingsConnectedViewModel.swift

```diff
@@ -0,0 +1,62 @@
+import Foundation
+import Yosemite
+
+final class CardReaderSettingsConnectedViewModel: CardReaderSettingsPresentedViewModel {
+
+    var shouldShow: CardReaderSettingsTriState
+    var didChangeShouldShow: ((CardReaderSettingsTriState) -> Void)?
+
+    private var didGetConnectedReaders: Bool
+    private var connectedReaders: [CardReader]
```

Comment:
It would be possible to initialise these properties here, next to their declaration:
`private var didGetConnectedReaders = falsea
`private var connectedReaders = []`

And remove the initialisation from the constructor. I am not sure if that’s better though 😊

Reply:
Sure! I'll change `CardReaderSettingsUnknownViewModel` that way too. I have no strong feelings either way

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/4108/commits/2e04020badf09266341640b9150430f168b126bb